app/src/main/java/com/eventyay/organizer/utils/CurrencyUtils.java

```diff
@@ -72,12 +72,16 @@ public CurrencyUtils() {
         countryCurrencyMap = new TreeMap<>();
         for (Locale locale : Locale.getAvailableLocales()) {
             try {
-                Currency currency = Currency.getInstance(locale);
                 String country = locale.getDisplayCountry();
-                if (currency != null && country != null && !country.equals("")) {
+                Currency currency = null;
+                if( !Utils.isEmpty(country)&&!locale.toString().matches(".*\\d+.*"))
+                    currency = Currency.getInstance(locale);
+                if (currency != null && country != null) {
```

Comment:
Why removed the empty check?

Reply:
https://github.com/fossasia/open-event-orga-app/pull/1440#issuecomment-454515293 
it works the same as you mentioned

Reply:
Sir as you mentioned in the above comment https://github.com/fossasia/open-event-orga-app/pull/1440#issuecomment-454515293 
**Use Utils.isEmpty instead** 
that's why , I have replaced it with Utils.isEmtpy() method

Reply:
You're not looking at the line I have commented on

Reply:
You're not able to look at line 80 and I don't know what to do with this PR now

Reply:
**if (currency != null && country != null)** are talking about this line please explain what is the problem ? Should I use Utils.isEmpty() method to check whether country string is null? Please clarify ? 
Or you mean that Exception handling used here is Unnecessary now ?
(Please help It is my second pull  request and I am new to Open source and github)

Reply:
Look at what it was before you changed it and yes, that is the line I commented on

Reply:
It was like that before https://github.com/fossasia/open-event-orga-app/blob/b2e22595e5ebe8bc1edd36ea23abe3d1cd495bc4/app/src/main/java/com/eventyay/organizer/utils/CurrencyUtils.java#L77
I just moved the check  !**country.equals("")** .
So should I use !Utils.isEmpty(country) instead instead of currency != null ?

Reply:
Yes